### ISSUE TITLE
Branch filter

### ISSUE THREAD

**Issue by andersforsgren** [cribeiro84/azure-devops-pull-request-hub#68]
I tried to filter the pull request list to use as an overview of active PRs for a team in a project with multiple teams. There doesn't seem to be a way of filtering on PR creator belonging to a certain team, so I tried using using the branch paths (Teams use topic/teamname/user/... prefixes for branches) so would like the source branches to be e.g. `feature/teamname1/`  or `feature/teamname1/*` to match PRs having any source branch matching the wildcard.

Would it be possible to make the branch filters work like branch entry does elsewhere in DevOps, i.e.  a path match with wildcards, such as target = version/*  (All pull requests for maintenance versions) or source branch = topic/teamX/*  (All pull requests for teamX)?

**Comment by cribeiro84** [comment]
Hello @andersforsgren 
Thanks for your feedback and inputs. This is a nice feature that I'll include in the future releases.

**Comment by cribeiro84** [comment]
@andersforsgren does this issue is still valid for you? We introduced filter by Tags where it could be easily used to fulfil your requirement. Would that work for you instead of this request by Branch name wildcards?

**Comment by andersforsgren** [comment]
It could but I'm not sure I understand how! :D
Do you mean the PR's would need to have manual tags (Labels) by team?

**Comment by cribeiro84** [comment]
@andersforsgren exactly.. then you would be able to add more tags as necessary. In meantime I'm checking how to get the wildcards using the existing components :-)
